x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -0,0 +1,197 @@
+  const { prepend } = core.http.basePath;
+  const { indexName, rangeFrom, rangeTo, kuery, status, setRefetch } = props;
+  const [flyoutAlert, setFlyoutAlert] = useState<TopAlert | undefined>(undefined);
+  const handleFlyoutClose = () => setFlyoutAlert(undefined);
+  const { timelines } = useKibana<{ timelines: TimelinesUIStart }>().services;
+
+  const leadingControlColumns = [
+    {
+      id: 'expand',
+      width: 40,
+      headerCellRender: () => null,
+      rowCellRender: ({ data }: ActionProps) => {
+        const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
+        const decoratedAlerts = decorateResponse(
+          [dataFieldEs] ?? [],
+          observabilityRuleTypeRegistry
+        );
+        const alert = decoratedAlerts[0];
+        return (
+          <EuiButtonIcon
+            size="s"
+            iconType="expand"
+            color="text"
+            onClick={() => setFlyoutAlert(alert)}
+          />
+        );
+      },
+    },
+    {
+      id: 'view_in_app',
+      width: 40,
+      headerCellRender: () => null,
+      rowCellRender: ({ data }: ActionProps) => {
+        const dataFieldEs = data.reduce((acc, d) => ({ ...acc, [d.field]: d.value }), {});
+        const decoratedAlerts = decorateResponse(
+          [dataFieldEs] ?? [],
+          observabilityRuleTypeRegistry
+        );
+        const alert = decoratedAlerts[0];
+        return (
+          <EuiButtonIcon
+            size="s"
+            target="_blank"
+            rel="nofollow noreferrer"
+            href={prepend(alert.link ?? '')}
+            iconType="inspect"
+            color="text"
+          />
+        );
+      },
+    },
+  ];
+
+  return (
+    <>
+      {flyoutAlert && (
+        <Suspense fallback={null}>
+          <LazyAlertsFlyout
+            alert={flyoutAlert}
+            observabilityRuleTypeRegistry={observabilityRuleTypeRegistry}
+            onClose={handleFlyoutClose}
+          />
+        </Suspense>
+      )}
+      {timelines.getTGrid<'standalone'>({
+        type: 'standalone',
+        columns,
+        deletedEventIds: [],
+        end: rangeTo,
+        filters: [],
+        indexNames: [indexName],
+        itemsPerPage: 10,
+        itemsPerPageOptions: [10, 25, 50],
+        loadingText: i18n.translate('xpack.observability.alertsTable.loadingTextLabel', {
+          defaultMessage: 'loading alerts',
+        }),
+        footerText: i18n.translate('xpack.observability.alertsTable.footerTextLabel', {
+          defaultMessage: 'alerts',
+        }),
+        query: {
+          query: `${ALERT_STATUS}: ${status}${kuery !== '' ? ` and ${kuery}` : ''}`,
+          language: 'kuery',
+        },
+        renderCellValue: getRenderCellValue({ rangeFrom, rangeTo, setFlyoutAlert }),
+        rowRenderers: NO_ROW_RENDER,
+        start: rangeFrom,
+        setRefetch,
+        sort: [
+          {
+            columnId: '@timestamp',
```

Comment:
This essentially means "sort by last update", which might not be intuitive to the user. How about we sort by "triggered" (`kibana.rac.alert.start`) instead?

```suggestion
            columnId: 'kibana.rac.alert.start',
```